CONTRIBUTORS_QUICK_START.rst

```diff
@@ -543,10 +543,18 @@ To avoid burden on CI infrastructure and to save time, Pre-commit hooks can be r
 
 1.  Installing required packages
 
+on Linux, install via
```

Comment:
```suggestion
on Debian / Ubuntu, install via
```

Reply:
Thank you for your suggestion, I committed it.